cf-oscore/src/main/java/org/eclipse/californium/oscore/OSCoreCtx.java

```diff
@@ -79,8 +80,47 @@
 
 	private Code CoAPCode = null;
 
-	//Include the context id in messages generated using this context
-	private boolean includeContextId;
+    /**
+     * Include the context id in messages generated using this context.
+     * This is generally optional and can be controlled by the application.
+     *
+     * Default value is false.
+     */
+    private boolean includeContextId;
```

Comment:
My editor shows
\s\s\s\s
should be
\t

Do you like to reformat it? Or should we just merge it, "as it is"?

Reply:
Fixed in commit 6b98ac703fd398530455b8c5ea3f9e617268fed0